src/scss/navigation/sidenav/_sidenav-primary.scss

```diff
@@ -0,0 +1,45 @@
+// 
+// Sidenav Primary
```

Comment:
This is a new file for the `sb-sidenav-primary` style, based on `sb-sidenav-dark`.